projects/packages/sandbox/license.txt

```diff
@@ -0,0 +1,357 @@
+This program is free software; you can redistribute it and/or modify
```

Comment:
We don't need this file. It's auto-generated and added to the mirror repo by way of the composer.json. I'm going to delete it and re-push to try to retrigger the checks.

Reply:
oh, wait... draft PR... ignore moe...